src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/external-references/GeneExternalReferences.tsx

```diff
@@ -113,15 +113,16 @@ const buildCrossReferenceGroups = (crossReferences: CrossReference[]) => {
 const GeneExternalReferences = () => {
   const params: EntityViewerParams = useParams();
 
-  const entityId = params.entityId?.split(':').pop();
+  const { entityId, genomeId } = params;
+
+  const stableId = entityId?.split(':').pop();
```

Comment:
there's a helper for this:

https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/state/ens-object/ensObjectHelpers.ts#L85-L91